src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exceptions.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationService REGISTRATION_SERVICE = new RegistrationServiceImpl();
+
+    @Test
+    void register_theSameLogin_NotOk() {
+        User testUser = new User();
+        testUser.setLogin("WormJim");
+        testUser.setPassword("123456");
+        testUser.setAge(29);
+        User testUser2 = new User();
+        testUser2.setLogin("WormJim");
+        testUser2.setPassword("654321");
+        testUser2.setAge(24);
+        User expected = REGISTRATION_SERVICE.register(testUser);
+        User actual = REGISTRATION_SERVICE.register(testUser2);
+        assertEquals(expected, actual, "Test failed you should not register the same user's login");
+    }
```

Comment:
In this test, you are asserting that two users with the same login can be registered, which contradicts the requirements in the checklist. The register method should not allow the registration of two users with the same login. You should instead assert that an exception is thrown when trying to register a user with a login that is already in use.